### ISSUE TITLE
Searching in UI with "All" option doesn't return anything

### ISSUE THREAD

**Issue by sajjadpz** [dice-group/autoindex#39]
Searching with label and `All` option selected returns nothing and throws exception
```
org.elasticsearch.index.IndexNotFoundException: no such index
```

It should search within all classifications e.g `class`, `property`, `entity` when `All` option is selected.

**Comment by prasanthhs** [comment]
@sajjadpz  I tried in my environment and it returns data. Can you confirm if the indexes were created? It may be possible that the indices don't exist and hence the problem.
![screen shot 2018-06-29 at 22 52 35](https://user-images.githubusercontent.com/33149420/42114415-4fbb2318-7bef-11e8-9c07-e4dc8bbe3d70.png)

**Comment by sajjadpz** [comment]
@prasanthhs - Indexes are created, I have tested with chrome-head plugin and search is also possible with `head-plugin`.

**Comment by prasanthhs** [comment]
@sajjadpz I have fixed this from code. Let me put it up. Basically when searching with "All", if index doesn't exist, we will create an empty index.

**Comment by prasanthhs** [comment]
Fixed in #40
